packages/plugins/plugin-table/src/components/ObjectDetailsPanel.tsx

```diff
@@ -23,9 +23,6 @@ const ObjectDetailsPanel = ({ objectId, view }: RowDetailsPanelProps) => {
   const space = getSpace(view);
   const schema = useSchema(client, space, view.query?.typename);
 
-  // TODO(burdon): Why is this needed?
-  const effectSchema = useMemo(() => schema?.snapshot, [JSON.stringify(schema?.jsonSchema)]);
-
```

Comment:
Pretty sure this was here because schema?.jsonSchema changes weren't reactive before. We should check that.

Reply:
@dmaretskyi how do we deal with reactivity?